README.MD

```diff
@@ -6,17 +6,10 @@ This repository contains the source code for the modelix website. The website is
 
 To be able to edit the website and deploy it you need to have two things installed:
 
- - hugo 0.110 (newer versions will break)
+ - hugo 0.128.2
```

Comment:
Version can later be updated with Renvovate together with the version in build.yaml and deploy.yaml ^^